server/services/placeOnReportService.test.ts

```diff
@@ -582,4 +583,70 @@ describe('placeOnReportService', () => {
       })
     })
   })
+
+  describe('getAssociatedStaffDetails', () => {
+    it('returns the correct response', async () => {
+      getAgency.mockResolvedValue({
+        agencyId: 'MDI',
+        description: 'Moorland (HMP & YOI)',
+        longDescription: 'HMP & YOI Moorland Prison near Doncaster',
+        agencyType: 'INST',
+        active: true,
+      })
+      const staffMembers = [
+        {
+          activeCaseLoadId: 'MDI',
+          email: 'testerPerson@justice.gov.uk',
+          firstName: 'Tester',
+          lastName: 'Person',
+          name: 'Tester Person',
+          staffId: 1234564789,
+          username: 'RO_USER_TEST',
+          verified: true,
+        },
+      ]
+      const response = await service.getAssociatedStaffDetails(staffMembers, user)
+      expect(response).toEqual([
+        {
+          activeCaseLoadId: 'MDI',
```

Comment:
Just a note for the future (and now if you want) - I think it is better to use the input as a basis for the expected output, if that is what is expected.

In this case, you have almost all these details in `staffMembers` - it is much less typing, and makes it clearer what you are testing, if you just inherit those values.